### PR TITLE
gphotos: new flag include-archived (#4728)

### PR DESCRIPTION
**Pull request by enere**
#### What is the purpose of this change?

new flag --gphotos-include-archived to download archived media when needed.

#### Was the change discussed in an issue or in the forum before?

suggested in #4728

#### Checklist

- [X] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [X] I have added tests for all changes in this PR if appropriate.
- [X] I have added documentation for the changes if appropriate.
- [X] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [X] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: backend/googlephotos/googlephotos.go [hunk 1]
```diff
@@ -132,15 +132,26 @@ you want to read the media.`,
 			Default:  2000,
 			Help:     `Year limits the photos to be downloaded to those which are uploaded after the given year`,
 			Advanced: true,
+		}, {
+			Name:    "include_archived",
+			Default: false,
+			Help: `Also download archived media.
+
+By default rclone do not request archived media. Thus when syncing,
```
Comment (ncw) [inline backend/googlephotos/googlephotos.go:140]: Apologies for this, but do you mind if I re-write this help a bit?

```
By default rclone does not request archived media. Thus, when syncing,
archived media is not visible in directory listings or transferred.

Note that media in albums is always visible and synced, no matter
their archive status.

With this flag, archived media are always visible in directory
listings and transferred.

Without this flag, archived media will not be visible in directory
listings and won't be transferred.
```

File: backend/googlephotos/googlephotos.go [hunk 2]
```diff
@@ -132,15 +132,26 @@ you want to read the media.`,
 			Default:  2000,
 			Help:     `Year limits the photos to be downloaded to those which are uploaded after the given year`,
 			Advanced: true,
+		}, {
+			Name:    "include_archived",
+			Default: false,
+			Help: `Also download archived media.
```
Comment (ncw) [inline backend/googlephotos/googlephotos.go:138]: This should probably say

`Also view and download archived media`

**Review by ncw — COMMENTED** [review]
This looks great - thank you :-)

Inline I've put some suggestions for the help text to make it read better.

Thanks!

File: backend/googlephotos/googlephotos.go (hunk 1 above)
Comment (enere) [reply backend/googlephotos/googlephotos.go]: Nice, I included your suggestion in my pull request.

File: backend/googlephotos/googlephotos.go (hunk 2 above)
Comment (enere) [reply backend/googlephotos/googlephotos.go]: Nice, pull request updated, thanks.

**Review by ncw — APPROVED** [review]
That looks great now - thank you very much :-)

I'll merge it now

**Pull request merged**